run_eval.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import json
+import lm_eval
+
+from hugging_face_model import RetNetModelHF, TransformerModelHF
+from pathlib import Path
+from torchscale.architecture.config import RetNetConfig, DecoderConfig
+from transformers import AutoModel, AutoConfig, AutoModelForCausalLM
+from typing import Optional, List
+
+def run_eval(model_path_dir: str, tokenizer_path_dir: Optional[str] = None, tasks: List[str] = ['hellaswag'], device: str = 'cuda:0', results_out_path: Optional[str] = None):
```

Comment:
Please alphabetize the parameters (it is fine to keep model_path_dir at beginning as it has no default.)